alf/algorithms/config.py

```diff
@@ -86,32 +79,34 @@ def __init__(self,
             num_eval_episodes (int) : number of episodes for one evaluation
             summary_interval (int): write summary every so many training steps
             update_counter_every_mini_batch (bool): whether to update counter
-                for every mini batch. The `summary_interval` is based on this
+                for every mini batch. The ``summary_interval`` is based on this
                 counter. Typically, this should be False. Set to True if you
                 want to have summary for every mini batch for the purpose of
-                debugging.
+                debugging. Only used by ``OffPolicyAlgorithm``.
```

Comment:
Instead of adding this sentence to every argument that is only used by OffPolicyAlgorithm, should we add a short prefix/suffix to the argument name to indicate this?

Reply:
I think it's okay. Not that many options are only used by OffPolicyAlgorithm